<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWindow</class>
 <widget class="QDialog" name="addWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>520</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>520</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Script MT</family>
     <pointsize>30</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add A New Resource</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Following Information:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Go Back To The Main Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset All Enterred Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>560</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add A New Resource</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>580</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(For Existing Resources The Qunatity Will Be Incremented)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>65</y>
     <width>391</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Resource's Title:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputTitle"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Resource's Author:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputAuthor"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Resource's Publication Year:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputPublicationYear"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Resource's Quantity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputQuantity"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Resource's Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="type">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="book">
         <property name="text">
          <string>Book</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="article">
         <property name="text">
          <string>Article</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="digitalcontent">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Digital Content</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="other">
         <property name="text">
          <string>Other</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
